### PR TITLE
7 5/add code embeds

### PR DESCRIPTION
**Pull request by tanvibhakta**
This PR will close https://linear.app/issue/NUWEB-150/[gutenberg]-add-support-for-code-blocks

### CODE REVIEW

File: src/components/Code.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from "react"
+import { PrismLight as SyntaxHighlighter } from "react-syntax-highlighter"
+import kotlin from "react-syntax-highlighter/dist/esm/languages/prism/kotlin"
+import { okaidia } from "react-syntax-highlighter/dist/esm/styles/prism"
+
+SyntaxHighlighter.registerLanguage("kotlin", kotlin)
```
Comment (meetbhatt23) [inline src/components/Code.js:6]: Any specific reason why this needs to be done?

File: src/components/BodyText.js [hunk 2]
```diff
@@ -54,6 +54,13 @@ export const BodyText = ({ children, className, content, type }) => {
         </span>
       )
 
+    case "code":
```
Comment (meetbhatt23) [inline src/components/BodyText.js:57]: Is this being used anywhere?

File: src/components/Code.js (hunk 1 above)
Comment (tanvibhakta) [reply src/components/Code.js:6]: Yes - when using the 'Light' version of the embed, the languages aren't registered by default, which means that one needs to individually register them.
explained in the docs here: https://github.com/conorhastings/react-syntax-highlighter/blob/master/README.md#light-build

File: src/components/BodyText.js (hunk 2 above)
Comment (tanvibhakta) [reply src/components/BodyText.js:57]: nope - left over from first commit. Good catch! Thanks :)

File: src/components/Code.js (hunk 1 above)
Comment (meetbhatt23) [reply src/components/Code.js:6]: Sure, but then this only works for `kotlin` then. Can we do the imports dynamically?

**Comment by meetbhatt23** [comment]
@tanvibhakta merging this for now but let's put the dynamic import into an icebox issue?

**Pull request merged**